CMakeLists.txt

```diff
@@ -139,7 +140,7 @@ execute_process(
 if(DEFINED ENV{SDK_VERSION})
   set(OSQUERY_BUILD_SDK_VERSION "${ENV{SDK_VERSION}}")
 else()
-  string(REPLACE "-" ";" OSQUERY_BUILD_SDK_VERSION ${OSQUERY_BUILD_VERSION})
+  string(REPLACE "-" ";" OSQUERY_BUILD_SDK_VERSION "1.4.5")
```

Comment:
Yes, I can easily add -DFOO=bar to CMAKE_ARGS for anything.

The port is aware of its version, so I can literally do something like:
CMAKE_ARGS+=-DOSQUERY_EXTERNAL_VERSION="${PORTVERSION}"